docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -569,6 +569,442 @@ export class Grid extends SmartContract {
+
+To send a transaction, create it as before and then use `tx.send()`:
+
+```ts
+// set Mina instance
+const Network = Mina.Network('https://example.com/graphql');
+Mina.setActiveInstance(Network);
+
+// create the transaction, add proofs and signatures
+const tx = await Mina.transaction(sender, () => {
+  // ...
+});
+await tx.prove();
+tx.sign([senderKey]);
+
+// send transaction
+await tx.send();
+```
+
+The output of `tx.send()` can be used to wait for inclusion of this transaction in a block, and to get the transaction hash (which lets you look up the pending transaction on a block explorer):
+
+```ts
+// send transaction, log transaction hash
+let pendingTx = await tx.send();
+console.log(`Got pending transaction with hash ${pendingTx.hash()}`);
+
+// wait until transaction is included in a block
+await pendingTx.wait();
+
+// our accounts updates are applied on chain!
+```
+
+Apart from `Mina.Metwork`, you can also use a mocked "Mina instance" for local testing:
+
+```ts
+const Local = Mina.LocalBlockchain();
+Mina.setActiveInstance(Local);
+```
+
+Doing this means setting up a fresh, local ledger, which is pre-filled with a couple of accounts with funds on them that you have access to. "Sending" a transaction here just means applying your account updates to that local Mina instance. This is helpful for testing, especially because account updates go through the same validation logic locally that they would on-chain.[^2]
+
+<!-- TODO commenting this out as it overlaps
+
+Everything else works the same locally as it would when interaction with a real network. You still need to properly sign your transactions and attach proofs, and then send them with `tx.send()` to apply them locally.
+
+To help you iterate quicker when testing, there's an option to disable proof creation: `Mina.LocalBlockchain({ proofsEnabled: false })`. You can use it like this:
+
+```ts
+// local mock Mina instance
+const proofsEnabled = false;
+const Local = Mina.LocalBlockchain({ proofsEnabled });
+Mina.setActiveInstance(Local);
+
+// only need to compile smart contract when proofs are enabled
+if (proofsEnabled) await MyContract.compile();
+
+const tx = await Mina.transaction(sender, () => {
+  // ...
+});
+await tx.prove();
+tx.sign([senderKey]);
+
+await tx.send();
+
+// our accounts updates are applied locally!
+```
+
+Note that with `proofsEnabled: false`, you still need to call `await tx.prove()`. The only change is that `.prove()` creates a dummy proof instead of a real one, and runs in 0 seconds. Also, `tx.send()` will skip the usual step of verifying your proof. -->
+
+You can learn more about testing in [How to test your zkApp](how-to-test-a-zkapp).
+
+[^2]: Fun fact: `LocalBlockchain` literally uses the same OCaml code for transaction validation and application that the Mina node uses; it's compiled to JS with [js_of_ocaml](https://github.com/ocsigen/js_of_ocaml).
```

Comment:
This is awesome, I like the idea of adding small fun facts and good-to-know things!